src/aat/resources/features/F-064/F-064.feature

```diff
@@ -0,0 +1,54 @@
+@F-064
+Feature: F-064: Retrieve workbasket input details for dynamic display
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-217
```

Comment:
The scenario is not matching with the identified scenario in latest excel sheet. 

S-217 | CCD Data Store Api :: Retrieve   workbasket input details for dynamic display | 401: Unauthorised | must return 401 when request   does not provide valid authentication credentials
-- | -- | -- | --